pombola/south_africa/management/commands/south_africa_mirror_mapit_wards.py

```diff
@@ -0,0 +1,64 @@
+from contextlib import closing
+import os
+from tempfile import NamedTemporaryFile
+import time
+from urlparse import urljoin
+
+import requests
+
+from django.contrib.gis.gdal import DataSource
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from mapit.models import Area, Country, Geometry, Generation, Type
+from mapit.management.command_utils import save_polygons
+
+SLEEP_BETWEEN_REQUESTS = 1
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        remote_mapit_url = 'http://mapit.code4sa.org/'
+        all_wards_url = urljoin(remote_mapit_url, '/areas/WD')
+        r = requests.get(all_wards_url)
+        time.sleep(SLEEP_BETWEEN_REQUESTS)
+        with transaction.atomic():
+            # Remove existing wards from MapIt:
+            Geometry.objects.filter(area__type__code='WD').delete()
+            Area.objects.filter(type__code='WD').delete()
+            # Get the Country object for South Africa:
```

Comment:
It might be useful to have an option to not do this and just download missing ones for occasions where the import fails half way through.